src/main/scala/com/datastax/gatling/plugin/model/DseCqlStatements.scala

```diff
@@ -15,53 +15,38 @@ import io.gatling.commons.validation._
 import io.gatling.core.session.{Session, _}
 
 import scala.collection.JavaConverters._
+import scala.collection.mutable.ArrayBuffer
 import scala.util.{Try, Failure => TryFailure, Success => TrySuccess}
 
 
-trait DseCqlStatement {
-  def apply(gatlingSession: Session): Validation[Statement]
-}
-
-
-/**
-  * Apply Simple CQL Query String
-  *
-  * @param string CQL Query String
-  */
-case class DseCqlStringStatement(string: Expression[String]) extends DseCqlStatement {
-  def apply(gatlingSession: Session): Validation[SimpleStatement] = {
-    string(gatlingSession).flatMap(stmt => new SimpleStatement(stmt).success)
-  }
+trait DseCqlStatement extends DseStatement[Statement] {
+  def buildFromFeeders(session: Session): Validation[Statement]
```

Comment:
What's the rationale around `buildFromFeeders` instead of `apply` or `buildFromSession`? We don't necessarily have to use a feeder, direct manipulation of the session is possible.

Reply:
It makes sense, let me rename that :-)